deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -361,17 +361,17 @@ cephClusterSpec:
     # deviceFilter:
     # config:
     #   crushRoot: "custom-root" # specify a non-default root label for the CRUSH map
-    #   metadataDevice: "md0" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
+    #   metadataDevice: "nvme0n1" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
```

Comment:
The metadataDevice name would depend on the environment, not sure if this will help clarify for others.

Reply:
@Underknowledge It's better to keep it as is. Although your environment has a filesystem in md0, it's not in other environments. Likewise, nvme0n1 might and might not be used depends on environment.

Reply:
the change is the other way around. 
Is it possible to use a md0 disk as an block db device?

Reply:
Yes, it's suported. More precisely, LVM logical volume is supported. In this case, the device name will be mdX. In addition, setting LV as metadatadevice is a common case.